Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Serialization/EditContext.h>
+
+#include <Source/StaticRigidBodyComponent.h>
+#include <Source/EditorStaticRigidBodyComponent.h>
+
+namespace PhysX
+{
+    void EditorStaticRigidBodyComponent::Reflect(AZ::ReflectContext* context)
+    {
+        auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
+        if (serializeContext)
```

Comment:
suggestion: Could limit the scope of `serializeContext` to where it's valid to be used

```
if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
    serializeContext)

// or
if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
```

(same for `EditContext`)

Reply:
Yeah, I normally do this one like that, but it came from a copy/paste. I'll change it.